llama/patches/0018-fix-clip-compiler-error.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: jmorganca <jmorganca@gmail.com>
+Date: Tue, 25 Feb 2025 19:14:51 -0800
+Subject: [PATCH] fix-clip-compiler-error
```

Comment:
This feels like it's likely some prior clip.cpp patch that breaks something, so we might want to rebase this into the prior patche(s) that touch clip.cpp so that as you replay the patches one at a time, each one builds properly.  This should help future rebasing.

Reply:
I believe it's like this in upstream – I'll make a note to create a patch for this though